packages/@react-spectrum/datepicker/src/utils.ts

```diff
@@ -55,8 +58,22 @@ export function useVisibleMonths(maxVisibleMonths: number) {
 }
 
 function getVisibleMonths(scale) {
+  if (typeof window === 'undefined') {
+    return 1;
+  }
   let monthWidth = scale === 'large' ? 336 : 280;
   let gap = scale === 'large' ? 30 : 24;
   let popoverPadding = scale === 'large' ? 32 : 48;
   return Math.floor((window.innerWidth - popoverPadding * 2) / (monthWidth + gap));
 }
+
+export function useFocusManagerRef(ref: FocusableRef<HTMLElement>) {
```

Comment:
do we want to put in an options object for focusFirst/focusLast, tabbable true/false?

not sure where else might use this, buttongroup/actiongroup?

either way, allowing for something before the ref since that's usually the last arg we pass

Reply:
This is internal to the datepicker package at the moment so should be fine

Reply:
ah yep, saw the export and forgot to check if it was forwarded all the way out